QueryDemo.ipynb

```diff
@@ -0,0 +1,3670 @@
+       " 183  https://s3.amazonaws.com/bdnyc/nir_spectra/j05...              None   \n",
+       " 184  https://s3.amazonaws.com/bdnyc/nir_spectra/U20...              None   \n",
+       " 185  https://s3.amazonaws.com/bdnyc/SpeX/Prism/U132...              None   \n",
+       " \n",
+       "                                         local_spectrum   regime telescope  \\\n",
+       " 0    $BDNYC_spectra/optical_spectra/2MASS1538-1953_...  optical    Keck I   \n",
+       " 1               $BDNYC_spectra/nir_spectra/U20001.fits      nir      IRTF   \n",
+       " 2               $BDNYC_spectra/nir_spectra/U10088.fits      nir      IRTF   \n",
+       " 3               $BDNYC_spectra/nir_spectra/U11343.fits      nir      IRTF   \n",
+       " 4     $BDNYC_spectra/nir_spectra/U10170_0310-2756.fits      nir      IRTF   \n",
+       " ..                                                 ...      ...       ...   \n",
+       " 181             $BDNYC_spectra/nir_spectra/U10316.fits      nir      IRTF   \n",
+       " 182   $BDNYC_spectra/nir_spectra/U11076_1231+4959.fits      nir      IRTF   \n",
+       " 183  $BDNYC_spectra/nir_spectra/j0534-0631prismtc1....      nir      IRTF   \n",
+       " 184             $BDNYC_spectra/nir_spectra/U20623.fits      nir      IRTF   \n",
+       " 185              $BDNYC_spectra/SpeX/Prism/U13209.fits      nir      IRTF   \n",
+       " \n",
+       "     instrument     mode observation_date comments reference other_references  \n",
+       " 0         LRIS  Missing       2010-07-17     None    Cruz18             None  \n",
+       " 1         SpeX    Prism       2003-09-03     None    Cruz18             None  \n",
+       " 2         SpeX    Prism       2004-11-08     None    Cruz18             None  \n",
+       " 3         SpeX    Prism       2003-09-04     None    Cruz18             None  \n",
+       " 4         SpeX    Prism       2011-12-08     None    Cruz18             None  \n",
+       " ..         ...      ...              ...      ...       ...              ...  \n",
+       " 181       SpeX    Prism       2004-11-08     None    Cruz18             None  \n",
+       " 182       SpeX    Prism       2011-12-08     None    Cruz18             None  \n",
+       " 183       SpeX    Prism       2006-12-08     None    Cruz18             None  \n",
+       " 184       SpeX    Prism       2003-09-03     None    Cruz18             None  \n",
+       " 185       SpeX    Prism       2003-08-12     None    Cruz18             None  \n",
+       " \n",
+       " [186 rows x 12 columns]}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "db.search_string('Cruz18', fuzzy_search=False)  # search for strings exactly matching Cruz19 anywhere in the database\n",
+    "db.search_string('Cruz18', fuzzy_search=False, fmt='pandas')  # as above, but have each table as a pandas dataframe"
```

Comment:
I don't think we need to show both results, maybe just the one with the `fmt` parameter passed in.